### ISSUE TITLE
Key Events broken

### ISSUE THREAD

**Issue by ghost** [DeXP/nuklear_cross#2]
I don't know whether this is GDIP specific, but NKC_EKEY is broken.
With the NKC demo as a template I'm listening for a key in the main loop.
If I press a key it launches the program into 3 updates, all of type NKC_ENONE. Only if the key remains pressed, or if I repress it during those 3 updates, does it count as a NKC_EKEY.

As a result only "double taps" of a key get registered or holding the key down.

**Comment by DeXP** [comment]
Yes, it could be specific problem of GDI+. I did not used this event on GDI+ heavily...
I would have a look at it a bit later.

**Comment by DeXP** [comment]
The problem is probably somewhere in the event processing:
Function `nkc_poll_events`, call of `PeekMessageW`
https://github.com/DeXP/nuklear_cross/blob/master/nkc_frontend/nkc_gdip.h#L305
The event is just not replaced by `WM_KEYDOWN`.
